### ISSUE TITLE
calculation fields not initialized after update to CF 1.5.6.2

### ISSUE THREAD

**Issue by jpgoem** [CalderaWP/Caldera-Forms#2076]
## Do You Need Immediate Help?
No, because I can fall back to an earlier release

## Is This A Feature Request?
No

## What Version Of Caldera Forms, WordPress and PHP Are You Using?
WordPress Version: 4.8.3PHP Version: 7.1.10MySQL Version: 5.5.5Caldera Forms Version: 1.5.6.2 (and later)

## Does Your Issue Persist When You Disable All Other Plugins and Switch To The Default Theme?
Yes

## What Is The Unexpected Behaviour?
In a form with 2 fields (1. first field is a dropdown or other select field with each option having a numeric value and a label, **but no calculation value**, and a default option), and 2. second field is a calculation field, with value equal to the first field).

When viewing the form in CF 1.5.6.1 and earlier, the second field is initialized to the value of the first field, as expected.

When viewing the form in CF 1.5.6.2 or later, the second field is zero. You need to toggle the dropdown field first in order to get a value for the calculation field.

## What PHP Errors Have You Logged While Reproducing This Bug?
No

## What JavaScript Errors Have You Seen While Reproducing This Bug?
No

Sample form testform attached.
[testform-export.zip](https://github.com/CalderaWP/Caldera-Forms/files/1463567/testform-export.zip)

**Comment by Shelob9** [comment]
@jpgoem Is this still an issue in 1.5.7.1 for you?

Also, you said your fields have "no calculation value". Why not set the right value for calculation value?

**Comment by jpgoem** [comment]
Yes, the problem is still there in 1.5.7.1.  My fields have "no calculation value", because the form was created at the time (CF 1.4.8) when there was no Calculation Value setting. 

As you suggest, setting the right value for Calculation Value indeed solves the problem (in the 3 releases that I tested : 1.5.6.2 and later) !  So my problem is now solved.

But I am now curious to understand the difference between the Value setting and the Calculation Value setting. When to use the one versus the other, or both ? I searched in the documentation  (https://calderaforms.com/doc/select-options/), but I didn't find  the answer.
